corehq/messaging/smsbackends/mach/models.py

```diff
@@ -102,6 +102,6 @@ def send(self, msg, *args, **kwargs):
             params['msg'] = msg.text.encode('utf-16-be').encode('hex')
```

Comment:
Outside the scope of this PR, but noticed that this will fail in Py3.

Reply:
How so?

Reply:
Py2:
```python
>>> u'foo'.encode('utf-16-be').encode('hex')
'0066006f006f'
```

Py3:
```python
>>> u'foo'.encode('utf-16-be').encode('hex')
Traceback (most recent call last):
  File "<ipython-input-2-6be281f43817>", line 1, in <module>
    u'foo'.encode('utf-16-be').encode('hex')
AttributeError: 'bytes' object has no attribute 'encode'

>>> import binascii
>>> binascii.hexlify(u'foo'.encode('utf-16-be'))
b'0066006f006f'
```